geomstats/information_geometry/fisher_rao_metric.py

```diff
@@ -0,0 +1,157 @@
+"""Class to implement simply the Fisher-Rao metric on information manifolds."""
+
+from scipy.integrate import quad_vec
+
+import geomstats.backend as gs
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+
+class FisherRaoMetric(RiemannianMetric):
+    r"""Class to derive the information metric from the pdf in InformationManifold.
+
+    Given a statistical manifold with coordinates :math:`\theta`,
+    the Fisher information metric is:
+    :math:`g_{j k}(\theta)=\int_X \frac{\partial \log p(x, \theta)}{\partial \theta_j}
+        \frac{\partial \log p(x, \theta)}{\partial \theta_k} p(x, \theta) d x`
+
+    Attributes
+    ----------
+    information_manifold : InformationManifold object
+        Riemannian Manifold for a parametric family of (real) distributions.
+    support : list, shape = (2,)
+        Left and right bounds for the support of the distribution.
+        But this is just to help integration, bounds should be as large as needed.
+    """
+
+    def __init__(self, information_manifold, support):
+        dim = information_manifold.dim
```

Comment:
we may not need to define this variable. just to below `shape=(information_manifold.dim,)`.

Reply:
Done.